.github/fixtures/test-fixtures-locally.sh

```diff
@@ -17,6 +17,18 @@ git init
 # Commit
 "$FIXTURES_DIR/commit.sh"
 
+# todo: add windows detection script
+# If running on Windows, convert the path to Windows format
+# WARNING: make sure you have LF line endings in your scripts (not CRLF)
+#          as this can cause issues with 'diff' command
+#SCRIPT_DIR=$(cygpath -w "$SCRIPT_DIR")
+#FIXTURES_DIR=$(cygpath -w "$FIXTURES_DIR")
+
 # Show results
 echo -e "\n---Run git-cliff---"
 cargo run --manifest-path "$SCRIPT_DIR/../../Cargo.toml" -- -vv --config "$FIXTURES_DIR/cliff.toml" "${@:2}"
+
+# Run again showing side-by-side diff
+diff --side-by-side --color \
+  <(cargo run --manifest-path "$SCRIPT_DIR/../../Cargo.toml" -- -vv --config "$FIXTURES_DIR/cliff.toml" "${@:2}") \
```

Comment:
We can assign the output to a variable instead of running it again here.

Reply:
https://github.com/orhun/git-cliff/pull/696/commits/362e91f54bf23c6963d8b153161bfba95eb00ed7